cub/benchmarks/bench/transform/babelstream.cu

```diff
@@ -0,0 +1,178 @@
+// SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <cub/device/device_copy.cuh>
+
+// %RANGE% TUNE_THREADS tpb 128:1024:32
+// %RANGE% TUNE_ALGORITHM alg 0:4:1
+
+#include <cub/device/dispatch/dispatch_transform.cuh>
+
+#include <nvbench_helper.cuh>
+
+#if CUB_DETAIL_COUNT(__CUDA_ARCH_LIST__) != 1
+#  error "This benchmark does not support being compiled for multiple architectures"
+#endif
+
+#if !TUNE_BASE
```

Comment:
suggestion: benchmark should work for multiple architectures:
```suggestion
#if !TUNE_BASE
#if CUB_DETAIL_COUNT(__CUDA_ARCH_LIST__) != 1
#  error "This benchmark does not support being compiled for multiple architectures"
#endif
```

Reply:
I agree, thx!